compiler/src/iree/compiler/Codegen/LLVMGPU/test/ROCDL/pipeline_vector_distribute_gfx942.mlir

```diff
@@ -1312,3 +1314,127 @@ module {
 // MEMORY-LABEL: func.func @attention_gather_k
 // MEMORY-COUNT-3: memref.alloc
 // MEMORY-NOT:     memref.alloc
+
+// -----
+
+#translation = #iree_codegen.translation_info<pipeline = LLVMGPUVectorDistribute workgroup_size = [256, 1, 1] subgroup_size = 64>
+#lowering_config = #iree_gpu.lowering_config<{reduction = [0, 0, 0, 0, 0, 512], workgroup = [1, 1, 1, 32, 0, 0]}>
+
+// {indexing_maps = [
+//   affine_map<(d0, d1, d2, d3, d4) -> (d0, d1, d2, d3)>,
+//   affine_map<(d0, d1, d2, d3, d4) -> (d0, d1, d4, d3)>,
+//   affine_map<(d0, d1, d2, d3, d4) -> (d0, d1, d2, d4)
+// ],
+// iterator_types = ["parallel", "parallel", "parallel", "reduction", "parallel"]
+// }
```

Comment:
Remove commented out code

Reply:
Hmmm I thought its useful to understand the lowering config dimensionality.
Otherwise it appears as a set of magic numbers that is not represented in linalg_ext.attention op.